src/chrome-finder.ts

```diff
@@ -202,7 +206,7 @@ function sort(installations: string[], priorities: Priorities) {
       .map(pair => pair.path);
 }
 
-function canAccess(file: string): Boolean {
+function canAccess(file?: string): Boolean {
```

Comment:
```suggestion
function canAccess(file: string | undefined): Boolean {
```

we always want them to pass a file argument, but it's OK if it's undefined :)